src/components/BookCardsPage.js

```diff
@@ -0,0 +1,85 @@
+import { books } from '../material/bookdata.json'
+import BookCard from './BookCard'
+import styled from 'styled-components/macro'
+import React, { useState } from 'react'
+
+export default function BookCardsPage() {
+  const [bookmarkedBooks, setBookmarkedBooks] = useState([])
+  const [booksShown, setBooksShown] = useState('all')
+
+  let bookmarkedBooksArray
+  function handleBookmarkClick(currentBook) {
+    if (bookmarkedBooks.includes(currentBook)) {
+      bookmarkedBooksArray = bookmarkedBooks.filter(
+        book => book !== currentBook
+      )
+    } else {
+      bookmarkedBooksArray = [...bookmarkedBooks, currentBook]
+    }
+    setBookmarkedBooks(bookmarkedBooksArray)
+  }
+
+  return (
+    <>
+      <CardspageLayout>
```

Comment:
Maybe either `CardPageLayout` or:
```suggestion
      <PageLayout>
```

Reply:
Thanks, i named it CardPageLayout now :-)